apache/metron
# METRON-1311 Service Check Should Check Elasticsearch Index Templates

The Service Check in Ambari does not validate that the Elasticsearch index templates have been installed.  Without these index templates bad things can happen.  For example, the Alerts UI will not be able to display any alerts.  With this PR, the Elasticsearch index templates that are installed by Ambari are checked as part of the Metron Service Check.

## Testing

1. Launch a Metron environment like Full Dev

1. Run the Metron Service Check and ensure that it succeeds.  

1. Click on Metron > Metron Indexing > Delete Elasticsearch Templates

1. Run the Metron Service Check and ensure that it fails with an error like the following.
    ```
    Missing Elasticsearch index template: name=yaf_index
    ```

1.  Click on Metron > Metron Indexing > Install Elasticsearch Templates.

1. Run the Metron Service Check and ensure that it succeeds.  

## Pull Request Checklist
